rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,497 @@
+    image for QC.
+
+  - As a user with a keyboard, I want to generate a gene x cell matrix
+    for biological analysis of my data and combination with sequencing
+    assays.
+
+## **Scientific "guardrails"**
+
+Image-based transcriptomics assays are still in active development, and
+benchmarking and characterization of these data types is in process. As
+such we expect to utilize community feedback to iterate on scientific
+quality of the pipelines, and will seek governance approval for the
+inclusion of data from these pipelines in release only when the
+technologies are more mature.
+
+Before large-scale ingress of these complex datasets is begun, the
+proposed integration should be reviewed against the HCA Science
+governance’s understanding of both the communities likely use of imaging
+data as well as the long-term costs of storage and transfer. Based on
+the intended investment for imaging, additional effort at this initial
+stage of the architecture may be called for to increase value or
+decrease cost.
+
+## **Detailed Design**
+
+### Data Specification
+
+A data specification consists of format specifications for any files
+(inputs and outputs) used by imaging assays, and HCA metadata that sits
+on top of these data and describes information about the samples and how
+the data were acquired. Formats and specifications listed below are
+being developed by *starfish*, OME, and HCA Metadata teams. Each have
+working prototypes supported by active development teams.
+
+#### Input Data Formats
+
+##### SpaceTx Format
+
+To prototype the use of imaging data in the DCP, the *starfish* and OME
+teams propose that the DCP use [<span class="underline">SpaceTx
+Format</span>](https://spacetx-starfish.readthedocs.io/en/latest/sptx-format/index.html),
+a format that groups and indexes a potentially very large number of two
+dimensional images, forming 5-dimensional tensors over (time, channel,
+z, y, x). These dimensions are fairly consistent across imaging domains
+and will likely generalize to other use cases. SpaceTx created this
+format because they were unable to find an existing format that
+satisfied our cloud compute and data visualization use cases. *starfish*
+and OME have since become aware of contemporaneous development of
+projects that aim to satisfy similar use cases and are working towards
+unification, where possible across scientific disciplines and languages.
+
+To avoid writing a large amount of microscope-specific code at the
+outset, *starfish* requests that users carry out basic pre-processing of
+their data to fix registration or chromatic errors. More details on the
+specific requests made of users can be found
+[<span class="underline">here</span>](https://drive.google.com/a/chanzuckerberg.com/open?id=1x516WMMWkEKKQwLrqqBsKmsPEDNjFYugqxm400D2NjY).
+The exact specification of the image data format will depend upon the
+delivery date that the DCP requests for this prototype.
```

Comment:
How do we recover provenance for the microscope-specific code and are there avenues for standardizing (even if stratified by microscope or company)?

Reply:
We're working to include these microscope-specific steps for the reasons you outline, and because we want to make sure to capture "actually-raw" data. 

Some of these steps can be standardized by requiring specific additional input data from users (e.g. a picture of the point spread function of each channel of a camera would enable standardized deconvolution). However, we need to do additional research to understand what kinds of data our data generators will be willing to provide. 

The best chance at standardization will come from commercialization, through which data is generated by a single device.

Reply:
For now, let's just make sure to describe where the user started and what they did thereafter which sounds like what you are focused on.